transactions/idTableStaking/admin/remove_unapproved_nodes.cdc

```diff
@@ -0,0 +1,25 @@
+import FlowIDTableStaking from 0xIDENTITYTABLEADDRESS
+
+// This transaction ends the staking auction, which refunds nodes 
+// with insufficient stake
+
+transaction(ids: [String]) {
+
+    // Local variable for a reference to the ID Table Admin object
+    let adminRef: &FlowIDTableStaking.Admin
+
+    prepare(acct: AuthAccount) {
+        // borrow a reference to the admin object
+        self.adminRef = acct.borrow<&FlowIDTableStaking.Admin>(from: FlowIDTableStaking.StakingAdminStoragePath)
+            ?? panic("Could not borrow reference to staking admin")
+    }
+
+    execute {
+        let approvedIDs: {String: Bool} = {}
+        for id in ids {
+            approvedIDs[id] = true
+        }
```

Comment:
Maybe just make the transaction parameter `{String: Bool}` instead of constructing it in the transaction